### PR TITLE
Bluetooth support on raspberypi3

### PR DESCRIPTION
**Pull request by telphan**
@agherzan @floion

### CODE REVIEW

**Comment by agherzan** [comment]
test this please

**Comment by agherzan** [comment]
LGTM

File: layers/meta-raspberrypi [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit dedd75f08ffe3360722ab761da18987aa681edd1
+Subproject commit 953dc5a406be6e1ad78c8aaf90df66ff4d7e5275
```
Comment (floion) [inline layers/meta-raspberrypi]: Are we not using the jethro branch for this layer?

Comment (floion) [inline layers/meta-raspberrypi]: Apparently the jethro branch is not receiving backports/additions so guess this will do for now

Comment (floion) [inline layers/meta-raspberrypi]: LGTM

**Comment by floion** [comment]
LGTM

**Pull request merged**